<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="P04284" name="uiTests.RegressionTestCSRUI" tests="10" failures="1" timestamp="2020-04-01T12:58:35 BST" time="123.300" errors="0">
  <testcase name="testUpdateBillingAddressManually" time="15.793" classname="uiTests.RegressionTestCSRUI"/>
  <testcase name="testContactPreferences" time="12.373" classname="uiTests.RegressionTestCSRUI"/>
  <testcase name="testEditCustomerDetails" time="3.252" classname="uiTests.RegressionTestCSRUI"/>
  <testcase name="testResetPassword" time="23.452" classname="uiTests.RegressionTestCSRUI"/>
  <testcase name="testUpdateBillingAddressByEircode" time="18.275" classname="uiTests.RegressionTestCSRUI"/>
  <testcase name="testChangeEmailAddress" time="20.443" classname="uiTests.RegressionTestCSRUI"/>
  <testcase name="testViewCustomerDetails" time="4.650" classname="uiTests.RegressionTestCSRUI"/>
  <testcase name="testSuccessfulLogin" time="3.383" classname="uiTests.RegressionTestCSRUI"/>
  <testcase name="testCustomerSearch" time="3.545" classname="uiTests.RegressionTestCSRUI"/>
  <testcase name="testTempUnlock" time="18.134" classname="uiTests.RegressionTestCSRUI">
    <failure type="java.lang.AssertionError" message="expected:&lt;false&gt; but was:&lt;true&gt;">
      <![CDATA[java.lang.AssertionError: expected:<false> but was:<true>
at org.junit.Assert.fail(Assert.java:89)
at org.junit.Assert.failNotEquals(Assert.java:835)
at org.junit.Assert.assertEquals(Assert.java:120)
at org.junit.Assert.assertEquals(Assert.java:146)
at uiTests.RegressionTestCSRUI.testTempUnlock(RegressionTestCSRUI.java:381)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testTempUnlock -->
</testsuite> <!-- uiTests.RegressionTestCSRUI -->
